server/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -117,7 +120,8 @@ public Content render(
                     program,
                     allPossibleProgramApplicationStatuses,
                     downloadModal.getButton(),
-                    filterParams),
+                    filterParams,
+                    request),
```

Comment:
The view should be updated too --> "Search by name, application id, phone number".

Reply:
@swatkat1 I'm doing that below on line 186. Or is there somewhere else you're thinking of that should be updated?

Reply:
I see its changed! Thanks